artifacts/manifests/k8s-staging-provider-gcp/OWNERS

```diff
@@ -0,0 +1,19 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES.
+approvers:
+- aojea
+- msau42
+- saad-ali
+- mattcary
+- leiyiz
+- sunnylovestiramisu
```

Comment:
this people is no longer active AFAIK, add myself, @bowei , @cheftako and @YifeiZhuang for the time being